docs/language/overview.md

```diff
@@ -1418,18 +1394,18 @@ is a Boolean comparison between the product `a*b` and `c`.
 ### 7.2 Search Logic
 
 The search patterns described above can be combined with other elements
-to a search expression comprised of "search terms" that may be combined
+of a search expression comprised of "search terms" that may be combined
 using Boolean logic.
```

Comment:
I find this sentence a bit confusing. How about this?

```
The search patterns described above can be combined with other "search terms"
using Boolean logic to form search expressions.
```

@mccanne: You might want to weigh in here.

Reply:
@nwt: That wording is fine with me, but I'll wait to see if @mccanne has input before pushing a change.

Reply:
+1 for @nwt suggestion